tests/cucumber/features/3d-editor/combine-materials.feature

```diff
@@ -1,3 +1,4 @@
+@ignore
```

Comment:
Let's make sure we have the task to enable this back in Jira, pls.

Reply:
https://exabyte.atlassian.net/browse/SOF-5465